pkg/metaserver/agent/pod/kubelet.go

```diff
@@ -30,23 +30,34 @@ type KubeletPodFetcher interface {
 	// GetPodList returns those latest pods, and podFilter is a function to filter a pod,
 	// if pod passed return true else return false, if podFilter is nil return all pods
 	GetPodList(ctx context.Context, podFilter func(*v1.Pod) bool) ([]*v1.Pod, error)
+
+	// GetPod returns Pod by UID
+	GetPod(ctx context.Context, podUID string) (*v1.Pod, error)
```

Comment:
KubeletPodFetcher is underlay interface to get k8s pod, only need provide get raw pod list function, and no need provide `GetPod` function to get pod by uid，the `GetPod` function should provide in `PodFetcher`